### DISCUSSION TITLE
Looking for beta testers 🙋‍♂️

### DISCUSSION THREAD

**Discussion by lukevella** [lukevella/rallly#509] [Announcements]
Hello everyone,

We've hit a new milestone recently by introducing [versions](https://github.com/lukevella/rallly/tags) and [releases](https://github.com/lukevella/rallly/releases). Along side this, comes a change in our release strategy. Instead of pushing updates straight to production, we will have a new beta environment where we can test out new features before they are released to the general public.

## Using the Open Beta

If you're interested in being a beta tester, here's what you need to know:

* The beta is open to everyone. Just go to https://beta.rallly.co
* The beta runs in a different environment to the production instance, so any accounts or polls created in one cannot be accessed by the other.
* The data in this environment can be deleted without any notice, so please **do not use it for anything important**.
* The beta environment has a **Feedback** menu item for ways on giving feedback.

![CleanShot 2023-02-10 at 17 59 05](https://user-images.githubusercontent.com/676849/218162910-e5375e96-1374-4772-9ef2-d1fcf3d646bb.png)

## What's new?

If you test out the beta environment now you will notice that there is a different workflow for voting on a poll, along with a more polished UI.

https://user-images.githubusercontent.com/676849/218172260-118c4eed-81a8-40fc-9ea4-033d63ac5bc4.mp4

The reason for the new workflow is because we're working towards being able to allow participants to enter an email address. This will allow us to send them a link back to the poll to edit their response. This mitigates a very common problem where participants get locked out of editing their response because of losing their guest session.

Please let me know if you have any questions or suggestions on how to make this better. Hope to hear from you very soon!

**Comment by iamericfletcher** [comment]
This is terrific! I would love to enter the beta environment whenever something is pushed to it. I've subscribed to this thread, any chance it could be updated when the beta receives updates?

**Reply by lukevella** [reply]
Thanks Eric. I think the best option to get updates right now is to look at the Github channel on discord. I will build something in to show updates within the app though. Still a work in progress 🙌

You might notice that the beta is very slow. This is because since it does not get much traffic, it suffers from cold starts on our serverless functions. Unfortunately there's not much that can be done about that at the moment but perhaps in the future we'll have it run in server environment to make it more usable.
